src/row.rs

```diff
@@ -312,7 +312,7 @@ impl<'stmt> Row<'stmt> {
     ///
     /// Returns an `Error::InvalidColumnName` if `idx` is not a valid column
     /// name for this row.
-    pub fn get_raw_checked<I: RowIndex>(&self, idx: I) -> Result<ValueRef<'_>> {
+    pub fn try_get_raw<I: RowIndex>(&self, idx: I) -> Result<ValueRef<'_>> {
```

Comment:
Hmm, for non-raw gets it's `Row::get` vs `Row::get_unwrap`. Maybe this should be `get_raw` and `get_raw_unwrap`?

Reply:
It means that `get_raw_checked` becomes `get_raw` and `get_raw` becomes `get_raw_unwrap` !

Reply:
Also:
https://docs.rs/postgres/0.18.1/postgres/row/struct.Row.html#method.try_get
https://docs.rs/sqlx/0.3.5/sqlx/row/trait.Row.html#method.try_get

Reply:
Not sure I follow.

And IDK if sqlx/postgres are relevant — I think being consistent with non-raw gets is more important than following the names other crates use.